### PR TITLE
+spring security

### PR DESCRIPTION
**Pull request by Vosker**

### CODE REVIEW

**Review by KostiantynVorobiov — APPROVED** [review]
Looks good!

File: src/main/java/com/dev/cinema/config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+    @Autowired
+    public void configureGlobal(AuthenticationManagerBuilder authenticationBuilder)
+            throws Exception {
+        authenticationBuilder.inMemoryAuthentication()
+                .passwordEncoder(getEncoder())
+                .withUser("USER").password(getEncoder().encode("1234")).roles("USER");
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/config/SecurityConfig.java]:
```suggestion
                .withUser("USER")
                .password(getEncoder()
                .encode("1234"))
                .roles("USER");
```

File: src/main/java/com/dev/cinema/model/dto/CinemaHallRequestDto.java [hunk 2]
```diff
@@ -1,6 +1,9 @@
 package com.dev.cinema.model.dto;
 
+import javax.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    @NotNull
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/model/dto/CinemaHallRequestDto.java]: can int be null?

File: src/main/java/com/dev/cinema/model/dto/UserRequestDto.java [hunk 3]
```diff
@@ -1,8 +1,14 @@
 package com.dev.cinema.model.dto;
 
+import com.dev.cinema.validators.ValidEmail;
+import javax.validation.constraints.NotNull;
+
 public class UserRequestDto {
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/model/dto/UserRequestDto.java:8]: what about password validation?

**Comment by RomanDubovskyi** [comment]
remember abut commit names, are yours informative? 
image

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request closed without merging**